.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+  # Execute a "nightly" build at 2 AM UTC 
+  - cron:  '0 2 * * *'
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Run on latest vcpkg 
+      shell: bash
+      run: |
+        mkdir C:/vcpkg-test
+        git clone https://github.com/microsoft/vcpkg  C:/vcpkg-test/vcpkg
+        C:/vcpkg-test/vcpkg/bootstrap-vcpkg.sh
+        C:/vcpkg-test/vcpkg/vcpkg.exe --overlay-ports=${GITHUB_WORKSPACE} install --triplet x64-windows ipopt-binary
```

Comment:
Don't we need to install yet a further library after ipopt in order to test the full functionality?

Reply:
Done in https://github.com/robotology/robotology-vcpkg-binary-ports/pull/5/commits/81a4a37dcf0685202dd0efc6e202f45d3cbfe85f .